spec/import_spec.rb

```diff
@@ -430,6 +430,24 @@ def upload_file!(name, ext = 'csv')
         upload_file!(:authors)
         expect(Author.count).to eq(2)
       end
+
+      context 'when the option before_import raises a ActiveAdminImport::Exception' do
+        let(:options) { { before_import: ->(_) { raise ActiveAdminImport::Exception, 'error message' } } }
+
+        it 'should show error' do
+          expect(page).to have_content I18n.t('active_admin_import.no_file_error')
+          expect(Author.count).to eq(0)
+        end
+      end
+
+      context 'when the option before_batch_import raises a ActiveAdminImport::Exception' do
+        let(:options) { { before_batch_import: ->(_) { raise ActiveAdminImport::Exception, 'error message' } } }
+
+        it 'should show error' do
+          expect(page).to have_content I18n.t('active_admin_import.no_file_error')
```

Comment:
@linqueta can u comment this
I18n.t('active_admin_import.no_file_error')
 is
 "Error: %{message}"

Reply:
expect(page).to have_content I18n.t('active_admin_import.no_file_error', message: 'error message')
maybe this is valid expection